components/interview/ActionButton.tsx

```diff
@@ -1,11 +1,30 @@
 import AddAPersonIcon from './icons/add-a-person-svg-react';
 import MenuNewSessionIcon from './icons/menu-new-session.svg.react';
+import AddNode from './icons/LucideAddNode';
+import type { NodeIcon } from '~/schemas/protocol/codebook/entities';
 
-export default function ActionButton({ onClick }: { onClick: () => void }) {
+const renderIcon = (icon: NodeIcon) => {
+  switch (icon) {
+    case 'add-a-person':
+      return <AddAPersonIcon />;
+    case 'add-a-place':
+      return <div>Place</div>; // TODO: implement add-a-place icon
+    default:
+      return <AddNode iconName={icon} />;
+  }
+};
+
+export default function ActionButton({
+  onClick,
+  icon,
+}: {
+  onClick: () => void;
```

Comment:
Props for this should probably just extend HTMLButtonElement, and then spread onto the button.